src/FakeItEasy/Sdk/Create.cs

```diff
@@ -0,0 +1,91 @@
+namespace FakeItEasy.Sdk
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel;
+    using System.Diagnostics.CodeAnalysis;
+    using System.Linq;
+    using FakeItEasy.Creation;
+
+    /// <summary>
+    /// Provides methods for generating fake objects.
+    /// </summary>
+    [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "A", Justification = "Is spelled correctly.")]
+    public static class Create
+    {
+        private static IFakeCreatorFacade FakeCreator
+        {
+            get { return ServiceLocator.Current.Resolve<IFakeCreatorFacade>(); }

```

Comment:
Making it a singleton doesn't seem to break anything. The same is true for `IFakeAndDummyManager` (it just breaks `ServiceLocatorTests`, which tests specifically that we get a new instance every time)